kevm-pyk/src/kevm_pyk/kproj/evm-semantics/optimizations.md

```diff
@@ -335,7 +370,8 @@ module EVM-OPTIMIZATIONS
       ...
     </kevm>
     requires ( #if USEGAS #then Gverylow < SCHED > <=Gas GAVAIL #else true #fi )
-     andBool ( #sizeWordStack( WS ) <=Int 1023 )
+     andBool ( 2 <=Int WSSize )
+     andBool ( WSSize <=Int 1023 )
```

Comment:
Oh, and the lower bound is different depending on the opcode.

Reply:
For example.